src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +18,6 @@ public static void main(String[] args) {
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
```

Comment:
It's good practice to handle possible exceptions that could be thrown when waiting for all futures to complete. Consider adding a try-catch block around CompletableFuture.allOf(futures).join() to handle potential InterruptedException or ExecutionException.